usagov/vote-gov-drupal
# Test/login script

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Insert Jira ticket link
](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-944&assignee=611272e0eb7aef00694b8a97)
## Description (optional)
This Pr will add log in scripts for different roles.  The login info is hosted in the `cyrpess.config.js` file and then the custom commands are in the `commands.js` file. The commands will log in before the test, will run the test and then will delete the user after as well. 

## Deployment and testing (required, if applicable)
### Testing steps
1. pull down branch 
2. start local 
3. cd into `testing` 
4. run `npm run cy:open` 
5. go to `backend` folder and there will be 3 tests (site admin, content editor and content manager)
6. ensure the tests pass and allow you to log in, right now it is just getting an element on the page.
